tests/functional/nightly/hosts

```diff
@@ -0,0 +1,12 @@
+[installer]
+client0 address=192.168.3.40
+
+[mons]
+mon0 address=192.168.3.10
+
+[osds]
+osd0 address=192.168.3.100
+
+[test_nodes:children]
```

Comment:
We need this to bootstrap the installer. Can you add a comment/note for this? Doesn't look like something that must be set always and it is critical this exists always.